superset/assets/src/components/AlteredSliceTag.jsx

```diff
@@ -56,13 +60,15 @@ export default class AlteredSliceTag extends React.Component {
       return 'N/A';
     } else if (value === null) {
       return 'null';
-    } else if (controls[key] && controls[key].type === 'FilterControl') {
+    } else if (controls[key] && controls[key].type === 'AdhocFilterControl') {
```

Comment:
was this line necessary before? Should I have already made this change or was it only necessary once we removed the legacy filters?

Reply:
@GabeLoins I'm not certain whether ad-hoc filters actually showed up in the altered slice view.